terraform/aks/variables.tf

```diff
@@ -0,0 +1,132 @@
+variable "app_environment" {
+  type = string
+}
+
+variable "app_suffix" {
+  type    = string
+  default = ""
+}
+
+variable "azure_resource_prefix" {
+  type = string
+}
+
+variable "azure_sp_credentials_json" {
+  type    = string
+  default = null
+}
+
+variable "cluster" {
+  type = string
+}
+
+variable "config_short" {
+  type = string
+}
+
+variable "deploy_azure_backing_services" {
+  type    = string
+  default = true
+}
+
+variable "docker_image" {
+  type = string
+}
+
+variable "enable_find" {
+  type    = bool
+  default = true
+}
+
+variable "enable_monitoring" {
+  type    = bool
+  default = true
+}
+
+variable "namespace" {
+  type = string
+}
+
+variable "service_short" {
+  type = string
+}
+
+variable "service_name" {
+  type = string
+}
+
+variable "key_vault_name" {
+  type = string
+}
+
+variable "key_vault_app_secret_name" {
+  type = string
+}
+
+variable "key_vault_infra_secret_name" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+# Below added to silence warnings
+variable "postgres_create_servicename_db" {
+  type    = string
+  default = "value"
+}
+
+variable "db_sslmode" {
+  type    = string
+  default = "value"
+}
+
+variable "env_config" {
+  type    = string
+  default = "value"
+}
+
+variable "postgres_extensions" {
+  type    = string
+  default = "value"
+}
+
+variable "postgres_version" { default = 13 }
+
+variable "app_config_file" {
```

Comment:
Is this file used anywhere?

Reply:
It's used here: https://github.com/DFE-Digital/publish-teacher-training/blob/9773bd3f9844af31c0d91c2751b22b9f0f89634d/terraform/aks/application.tf#L19

Reply:
Thanks